ethereum_history_api/circuits/lib/src/storage_test.nr

```diff
@@ -0,0 +1,50 @@
+use crate::storage::{verify_storage_value};
+use crate::fixtures::paris::usdc::storage_proof::storage_proofs_with_storage_root;
+
+#[test]
+fn test_verify_storage_proof() {
+    let _ = verify_storage_value(
+        storage_proofs_with_storage_root.storage_root,
+        storage_proofs_with_storage_root.proofs[0]
+    );
+}
+
+#[test(should_fail)]
+fn test_verify_storage_proof_invalid_storage_root() {
+    let mut storage_root = storage_proofs_with_storage_root.storage_root;
```

Comment:
let mut {storage_root} = storage_proofs_with_storage_root; (?)

Reply:
It can be done this way:
```
let mut StorageProofsWithStorageRoot { storage_root, proofs } = storage_proofs_with_storage_root;
```
but IMO this
```
let mut root = storage_proofs_with_storage_root.storage_root;
```
is more concise.